### ISSUE TITLE
vmalert generates alert with  empty .CommonLabels and/or CommonLabels.alertname

### ISSUE THREAD

**Issue by icy** [VictoriaMetrics/VictoriaMetrics#5330]
### Describe the bug

tl;dr: In alertmanager template, .CommonLabels and/or .CommonLabels.alertname is empty.

I'm using vmalert and vmalert-manager . I created this following rule

```
apiVersion: operator.victoriametrics.com/v1beta1
kind: VMRule
metadata:
  name: default
spec:
  groups:
  - name: devops-test
    rules:
    - alert: always_fired_alert
      expr: |
        1 < 0
      for: 1m
      labels:
        severity: warning
      annotations:
        description: |
          This is an always-fired-alert, to ensure that system is running well.
         runbook_url: "https://letmegooglethat.com/?q=google"
```

What I got from the alert with slack integration (default template) are some alerts with empty subject. I tried to investigate and found that the fields `.CommonLabels` and/or `CommonLabels.alertname` is empty. This doesn't happen with the first FIRING with a group of 1 alert, but it happens with multiple alerts and/or on subsequent FIRINGs:

```
# message got on Slack channel. My comment starts with #
[FIRING:1] always_fired_alert # good
[FIRING:3]                               # worse now...
[FIRING:3]                               # just empty alert info, no idea what's happened
[FIRING:3]
[FIRING:3]
[FIRING:3]
[FIRING:3]
``` 

All other fields (severity for example) are gone. However, the description field is still visible with proper template.

### To Reproduce

See above.

### Version

The version of alertmanager is 0.26.0, and vmalert and vmcluster version are all latest (v1.93.7-cluster).

**Comment by Haleygo** [comment]
Hello! @icy 
As it's name, `.CommonLabels` only contains the labels common to all of the alerts.
```
[FIRING:1] always_fired_alert # good
[FIRING:3]                               # worse now...
[FIRING:3]                               # just empty alert info, no idea what's happened
```
From your message here, you got one alert from rule `always_fired_alert`, so `CommonLabels` is the whole labelset of the rule.
Then you got three firing alerts which grouped by alertmanager according to `group_by` key. But rule `always_fired_alert` with expr `1 < 0` should only have one result series, the other two must come from other rules, so there is no commonLabel `alertname` or others between them thus empty message.

>What I got from the alert with slack integration (default template) are some alerts with empty subject.

What's your `group_by` key for slack [route](https://prometheus.io/docs/alerting/latest/configuration/#route) here? If it doesn't contain `alertname`, the grouped alerts could miss `CommonLabels.alertname`.

**Comment by icy** [comment]
Thanks @Haleygo for your explanation. Yes it was a wrong configuration, I totally forgot to update the `group_by` setting. Now it's working as expected.
